src/lib/PostgresMetaTriggers.ts

```diff
@@ -0,0 +1,230 @@
+
+      const { data, error } = await this.query(sql)
+
+      if (error) {
+        return { data: null, error }
+      }
+
+      const triggerRecord = data && data[0]
+
+      if (triggerRecord) {
+        return { data: triggerRecord, error: null }
+      }
+
+      return {
+        data: null,
+        error: {
+          message: `Cannot find a trigger with name ${name} on table "${schema}"."${table}"`,
+        },
+      }
+    }
+
+    return { data: null, error: { message: 'Invalid parameters on trigger retrieve' } }
+  }
+
+  /**
+   * Creates trigger
+   *
+   * @param {Object} obj - An object.
+   * @param {string} obj.name - Trigger name.
+   * @param {string} obj.schema - Name of schema that trigger is for.
+   * @param {string} obj.table - Unqualified table, view, or foreign table name that trigger is for.
+   * @param {string} obj.function_schema - Name of schema that function is for.
+   * @param {string} obj.function_name - Unqualified name of the function to execute.
+   * @param {('BEFORE'|'AFTER'|'INSTEAD OF')} obj.activation - Determines when function is called
+   * during event occurrence.
+   * @param {Array<string>} obj.events - Event(s) that will fire the trigger. Array of the following options: 'INSERT' | 'UPDATE' | 'UPDATE
+   * OF column_name1,column_name2' | 'DELETE' | 'TRUNCATE'.
+   * @param {('ROW'|'STATEMENT')} obj.orientation - Trigger function for every row affected by event or
+   * once per statement. Defaults to 'STATEMENT'.
+   * @param {string} obj.condition - Boolean expression that will trigger function.
+   * For example: '(old.* IS DISTINCT FROM new.*)'
+   * @param {string} obj.transition_relation_reference - Provide access to transition relations of triggering statement to function.
+   * For example: 'REFERENCING NEW TABLE AS newtable OLD TABLE AS oldtable'
+   * @param {Array<string>} obj.function_args - array of arguments to be passed to function when trigger is fired.
+   * For example: ['arg1', 'arg2']
+   */
+  async create({
+    name,
+    schema = 'public',
+    table,
+    function_schema = 'public',
+    function_name,
+    function_args,
+    activation,
+    events,
+    orientation,
+    condition,
+    transition_relation_reference,
+  }: {
+    name: string
+    table: string
+    function_name: string
+    activation: string
+    events: string[]
+    function_schema?: string
+    schema?: string
+    orientation?: string
+    condition?: string
+    transition_relation_reference?: string
+    function_args?: string[]
+  }): Promise<PostgresMetaResult<PostgresTrigger>> {
+    const qualifiedTableName = `${ident(schema)}.${ident(table)}`
+    const qualifiedFunctionName = `${ident(function_schema)}.${ident(function_name)}`
+    
+    const triggerOrientation = orientation ? `FOR EACH ${format.string(orientation)}` : ''
+    const triggerCondition = condition ? `WHEN ${format.string(condition)}` : ''
```

Comment:
Nitpick: I think it's better to omit the parens in the argument, i.e. treat it like `function_args`:
```suggestion
    const triggerCondition = condition ? `WHEN ( ${format.string(condition)} )` : ''
```